__tests__/app.test.js

```diff
@@ -0,0 +1,27 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const articleData = require("../db/data/test-data/articles");
+const commentData = require("../db/data/test-data/comments");
+const topicData = require("../db/data/test-data/topics");
+const userData = require("../db/data/test-data/users");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET: 200 responds with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const topicsArray = response.body.topics;
+        expect(Array.isArray(topicsArray)).toBe(true);
+        topicsArray.forEach((topic) => {
+          expect(topic).toHaveProperty("slug", expect.any(String));
+          expect(topic).toHaveProperty("description", expect.any(String));
+        });
```

Comment:
Using toHaveProperty isn't bad, but there is better Jest Matchers we can use for more robust and extendable testing, take a look into toMatchObjects and expect.objectContaining.